.gitattributes

```diff
@@ -1,7 +1,7 @@
 ###############################################################################
 # Set default behavior to automatically normalize line endings.
 ###############################################################################
-* text=auto
+#* text=auto
```

Comment:
why this change?

Reply:
I've tested a bit. Would you prefer to let it as before?

Reply:
yes, please revert if not really needed